### ISSUE TITLE
Resolvers: Default accepted_payload_size causes parsing error within HAProxy

### ISSUE THREAD

**Issue by dcorbett-haproxy** [haproxytech/dataplaneapi#89]
Hello,

When attempting to add a new resolvers if you only fill in the minimal required data then accepted_payload_size gets set to 0. HAProxy requires this to be between 12 and 8192. I look to @bedis for some guidance on how this should be handled but I believe this option should either remain unspecified if not specified by the user or default to the highest (8192).

To reproduce:
curl -H 'Content-Type: application/json' -u dataplaneapi:mypassword 'localhost:5555/v2/services/haproxy/configuration/resolvers?version=1' -d '{"name":"mydns", "parse-resolv-conf":true }'

The output from HAProxy will be:
[ALERT] 123/122923 (26598) : parsing [haproxy.cfg:31] : 'accepted_payload_size' must be between 12 and 8192 inclusive (was 0).
[ALERT] 123/122923 (26598) : Error(s) found in configuration file : haproxy.cfg
[ALERT] 123/122923 (26598) : Fatal errors found in configuration.

**Comment by mjuraga** [comment]
This is resolved with: 78b4820a99b9efdd0816c4c1a9a44e41ae884445

Also, there might be a bug in haproxy checking the value, the [doc](https://cbonte.github.io/haproxy-dconv/2.2/configuration.html#accepted_payload_size) states: 

```
If not set, HAProxy announces 512. (minimal value defined by RFC 6891)
```

So the `12` seems like the odd number here.

**Comment by bedis** [comment]
I guess this 12 comes from sizeof(struct(dns_header)) when we check if accepted-payload value provided by user.
it turns out that the code does not do what the doc says, so this is a bug :) I will fix this in upstream HAProxy.
